examples/basics/plotting/quick.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+# -----------------------------------------------------------------------------
+# Copyright (c) 2014, Vispy Development Team.
+# Distributed under the (new) BSD License. See LICENSE.txt for more info.
+# -----------------------------------------------------------------------------
+"""
+Example of plotting multiple things using the quick interface.
+"""
+
+import numpy as np
+
+from vispy.io import load_data_file
+import vispy.plot.q as vpq
```

Comment:
You are not alone. Here's an idea: what if we expose the CLI plotting interface via a completely separate top-level package?
```
import vplot as vp
vp.plot(...)
```
I seem to recall matplotlib doing something similar with `pylab`, is that right?